### ISSUE TITLE
Add page to show specific Resource or Link

### ISSUE THREAD

**Issue by cacarrara** [mochaoss/inspace#10]
Depends on: #9 

We need to have a page to show a specific Resource or Link

Url examples/suggestion:
```
http://inspace.herokuapp.com/resource/resource-slug
http://inspace.herokuapp.com/link/link-slug
```

The page design can follow the search result card:
![image](https://user-images.githubusercontent.com/348237/28398377-5b101a60-6cdd-11e7-924c-663dd02b5c88.png)

**Comment by Pal0r** [comment]
I addressed this [PR #25 ](https://github.com/mochaoss/inspace/pull/25), which updated links in the home page.

**Comment by cacarrara** [comment]
Done in #25
